main.js

```diff
@@ -41,9 +38,14 @@ app.on('window-all-closed', function () {
   if (process.platform !== 'darwin') app.quit()
 })
 
-function finishTest (success) {
-  process.exit(success ? 0 : 1)
-}
-function failTestIfBadProcess (details) {
-  if (details.reason !== 'clean-exit') finishTest(false)
+// Test helpers
+const test = {
+  done: (success) => process.exit(success ? 0 : 1),
+  init: () => {
+    crashReporter.start({ uploadToServer: false, submitURL: '' })
+    ipcMain.on('test-done', (_, success) => test.done(success))
+    const failIfBadExit = (details) => details.reason === 'clean-exit' || test.done(false)
```

Comment:
super nit-- this feels like a bash-ism, i think the following is a touch more readable
```suggestion
    const failIfBadExit = (details) => { if (details.reason !== 'clean-exit') test.done(false) }
```

Reply:
Fair enough. I think I was trying too hard to keep one property per line. Fixed in 9b3322369caab7af28e58c84726c2fed8c63eb42